### ISSUE TITLE
sendImage and sendFileFromUrl with waitForKey

### ISSUE THREAD

**Issue by JoeDozan24** [open-wa/wa-automate-nodejs#881]
**Describe the bug**
sendImage and sendFileFromUrl will not send response (the message ID). But with `waitForKey = false` still running well.

**Steps to Reproduce**
1. Start open-wa as usual
2. Do sendImage/sendFileFromUrl with `waitForKey = true`
3. Message sent (checked on WA mobile), but respond does not sent (open-wa)

**Expected behavior**
OPEN-WA send back the messageId of the sent message(s)

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
  WA_VERSION: '2.2039.9',
  PAGE_UA: 'WhatsApp/2.2037.6 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36',
  WA_AUTOMATE_VERSION: '2.0.25 UPDATE AVAILABLE: 2.0.26',
  BROWSER_VERSION: 'Chrome/80.0.3987.0'
}
```

**Host (please complete the following information):**
 - OS: Windows 10

**Comment by smashah** [comment]
thanks for the detail @JoeDozan24 will fix this asap

**Comment by smashah** [comment]
This was fixed in #752 via a patch
